scripts/dev-env-setup.sh

```
@@ -0,0 +1,29 @@
+
+echo "=== 開発環境の環境変数設定 ==="
+
+# Check if .env exists
+if [ ! -f .env ]; then
+    echo "❌ .envファイルが見つかりません"
+    echo "   scripts/setup-dev-secrets.sh を実行してください"
+    exit 1
+fi
+
+# Source .env file
+export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)